protobuf/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if (NOT EMIL_BUILD_PROTOBUF)
```

Comment:
I would rather prevent entering the subdirectory one level-up instead of this early return.